uber/templates/registration/new.html

```diff
@@ -120,6 +121,7 @@
 <table class="table datatable" data-page-length="-1">
 <thead>
     <tr>
+        <th align="left">Registered</th>
```

Comment:
The reason for this is that we want registrations to be sorted by when they registered by default.